cts.json

```diff
@@ -509,6 +509,26 @@
         }
       ]
     },
+    {
+      "name": "filter, existence without selector",
+      "selector": "$[?@]",
+      "document": {
+        "a": {},
+        "b": 1
+      },
+      "result": [
+        {},
+        1
+      ]
+    },
+    {
+      "name": "filter, existence test on primitive (not supported)",
```

Comment:
I think the wording "not supported" is a bit misleading. The wording in the standard is:

> Applied to a primitive value, it selects nothing [...]

Therefore, I suggest the test name:

```json
"filter, existence test on primitive, selects nothing"
```

I would apply this suggestion to other, similarly named tests.

Also, note that I did not include brackets, which seem to violate the general naming convention used throughout the test suite.